pyartcd/tests/pipelines/test_plashets.py

```diff
@@ -8,14 +8,14 @@ class TestBuildCompose(unittest.IsolatedAsyncioTestCase):
     @patch('pyartcd.jenkins.init_jenkins')
```

Comment:
While these tests are focused on the BuildPlashetsPipeline, it would be nice to cover the actual logic of `PlashetBuilder` itself.